packages/atomic/stencil-proxy/esm/loader.js

```diff
@@ -1,20 +1,7 @@
-import commerceComponents from '../atomic/components/components/commerce/lazy-index.js';
-import commonComponents from '../atomic/components/components/common/lazy-index.js';
-import insightComponents from '../atomic/components/components/insight/lazy-index.js';
-import ipxComponents from '../atomic/components/components/ipx/lazy-index.js';
-import recommendationsComponents from '../atomic/components/components/recommendations/lazy-index.js';
-import searchComponents from '../atomic/components/components/search/lazy-index.js';
 import {defineCustomElements as originalDefineCustomElements} from './_loader.js';
 
 const defineCustomElements = function (...args) {
-  Object.values({
-    ...commonComponents,
-    ...recommendationsComponents,
-    ...ipxComponents,
-    ...insightComponents,
-    ...commerceComponents,
-    ...searchComponents,
-  }).forEach((importFunction) => importFunction());
+  import('../atomic/autoloader/index.esm.js');
```

Comment:
We import dynamically to retain the 'pure' aspect of the module (i.e. importing it does not come with side effect)